test/seedFactoryV2.spec.js

```diff
@@ -21,7 +21,7 @@ const { SEVEN_DAYS } = require("./helpers/constants/time.js");
  * @typedef {import("../lib/types/types").Seed} Seed
  */
 
-describe("> Contract: SeedFactory", () => {
+describe("> Contract: SeedFactoryV2", () => {
```

Comment:
I'm wondering about the helpfulness of having test files for each contract version.
What would you do when there will be SeedFactoryV3?
Will you duplicate the test files?

I'm leaning towards, only one test suit for the latest version.

Every older version will then be considered unsupported? Wdyt?

Reply:
So the issue with having only one test file would be that the code coverage would then go to 0% for the older contract version, as soon as we switch the tests to a new major version. Or we decide to remove the outdated contracts from the repo and only have the ABIs in there?

One other option could be to add only tests for the change when a new version is developed. Although this would have the same issue with codecoverage

I do agree that it does not make sense to copy all the test code once a new version is being worked on.

I will remove the version from the test for now at least.

Reply:
We will need to brainstorm this further
Could you ask for input from your BE colleagues?

Reply:
Will do and keep you updated

Reply:
Altho I don't think it is very pressing in light of all the other things happening in the lab